src/test/garfile/customs/post.controller.test.js

```diff
@@ -185,4 +185,25 @@ describe('GAR Customs Post Controller', () => {
       expect(res.redirect).to.have.been.calledWith('/garfile/supportingdocuments');
     });
   });
+
+  it('should set goodsDeclaration=empty string if prohibited!=Yes', () => {
+    req.body.buttonClicked = 'Save and continue';
+    req.body.prohibitedGoods = 'No';
```

Comment:
If I had to be SUPER pedantic, set the goodsDeclaration to be some random string, so that the assertion of it being empty is setting it as empty despite the input. Probably do it an extra test to increase the test count!

Reply:
Done, added a unit test to cover the case with random string.